### PR TITLE
Add support for Arbitrum and send summary via slack

### PR DESCRIPTION
**Pull request by fleupold**
Most changes are coming from `cow-sdk` version bump in `yarn.lock` (needed to support Arbirum).

This PR does two things
1. Address #25 
2. Add support for Arbitrum

Also cleaned up some minor dead code (and made module mandatory, seems too brittle to rely on outdated deployments by defaults)

### Testplan

TODO: full local test-run on arbitrum

### CODE REVIEW

**Comment by mfw78** [comment]
Couldn't we just instead have some infrastructure-specific code that takes `stdout` as a summary (or otherwise uses `stderr` for communicating failure, and thus keep infrastructure separate from domain-specific logic?

**Comment by fleupold** [comment]
We could (in fact it was suggested in the original issue, but didn't receive much traction). My personal feeling is that given this contract is already very tailored to our domain, it's ok to add the slack integration here.

But can change if you fell strongly.

File: README.md [hunk 1]
```diff
@@ -53,7 +54,7 @@ variable.
 Usage: cow-fee [options]
 
 Options:
-  --network <network>                                   (choices: "mainnet", "gnosis")
+  --network <network>                                   (choices: "mainnet", "gnosis","arbitrum")
```
Comment (mfw78) [inline README.md:57]:
```suggestion
  --network <network>                                   (choices: "mainnet", "gnosis", "arbitrum")
```

**Review by mfw78 — COMMENTED** [review]
Was it intentional to redeploy on mainnet and gnosis?

**Comment by fleupold** [comment]
> Was it intentional to redeploy on mainnet and gnosis?

Yes, in order to include the new contracts that support withdrawing the to token

**Review by mfw78 — APPROVED** [review]

**Pull request merged**